### PR TITLE
feature(alert-button):Adding alert and button into component library

### PR DESCRIPTION
**Pull request by Chirag-kumar-dev**
##Description

feature(alert-button):Adding alert and button into component library

### CODE REVIEW

File: src/components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    color: white;
+    background-color: greenyellow;
+    border: 1px solid var(--rb-very-dark-green);
+  }
+  
+  .alert.warning {
+    color: white;
+    background-color: yellow;
+    border: 1px solid var(--rb-very-dark-brown);
+  }
```
Comment (vikrantkode) [inline src/components/alert/alert.css]: be specific as alert class is repeating. try not to repeat classes in CSS

**Review by vikrantkode — CHANGES_REQUESTED** [review]
UI can be improved 
- Footer is not uniform in all the components
- You can have a look again in avatar and badges
- Images can be small

**Comment by jatinmore** [comment]
-make some ui changes
-on docs page make sidebar and navbar fixed
-alert-button components are looking good to me
-avatar images could be smaller
-make sure to have consistentcy of existing elements/components in all the diff pages

File: src/components/alert/alert.css (hunk 1 above)
Comment (Chirag-kumar-dev) [reply src/components/alert/alert.css]: sure ,now alert is not repeating

**Review by rahul31agrawal — APPROVED** [review]

**Review by sohan9819 — APPROVED** [review]
Looks good to me

**Pull request merged**